src/app/components/Book/Book.jsx

```diff
@@ -6,12 +6,19 @@ import config from '../../../../appConfig';
 import utils from '../../utils/utils';
 
 const Book = ({ pick }) => {
-  const book = pick.book;
-  const tagArray = pick.tags.map(tag => tag.toLowerCase().split(' ').join('-'));
-  const tagClasses = tagArray.join(' ');
-
   const isStringEmpty = (string) => (!_isString(string) || _isEmpty(string.trim()));
 
+  const getBookObject = (obj) => (obj.book || {});
+
+  const getReviewsArray = (obj) => (obj.reviews || []);
+
+  const getTagsArray = (obj) => (
+    !_isEmpty(obj.tags) ? obj.tags.map(tag => tag.toLowerCase().split(' ').join('-')) : []
```

Comment:
Use the function that's in utils, `getPickTags`.

Reply:
Great, will do.